tests/integration/it_test_authorization.py

```diff
@@ -0,0 +1,111 @@
+import random
+import string
+from datetime import datetime
+
+import flask
+import pytest
+
+from timeless.access_control.authorization import is_allowed
+from timeless.access_control.methods import Method
+from timeless.companies.models import Company
+from timeless.employees.models import Employee
+
+
+# @todo #329:30min After #298 is finished update this test cases with
+#  appropriate roles and make sure that current test cases diffrentiate enough
+#  between roles, so we can be sure that is_allowed takes into account all
+#  exiting roles. Also remove pytest.mark.skip from all test methods here.
+def test_manager_can_access_employee(app, db_session):
+    my_company = Company(
+        name="Acme Inc.", code="code1", address="addr"
+    )
+    db_session.add(my_company)
+    db_session.commit()
+    me = create_employee(my_company, "manager")
+    db_session.add(me)
+    flask.g.user = me
+    other = create_employee(my_company, "master")
+    db_session.add(other)
```

Comment:
@krzyk May be better to do 
```
my_company = Company(
        name="Acme Inc.", code="code1", address="addr"
    )
me = create_employee(my_company, "manager")
flask.g.user = me
other = create_employee(my_company, "master")
db_session.add_all(my_company, me, other)
db_session.flush()
db_session.commit()
```
WDYT?

Reply:
@vladarefiev I need comany id when creating company, so the commit/flush needs to be before creating employee

Reply:
@krzyk Ah, ok